docs/source/format/Columnar.rst

```diff
@@ -21,7 +21,7 @@
 Arrow Columnar Format
 *********************
 
-*Version: 1.4*
+*Version: 1.5*
 
 .. seealso:: :ref:`Additions to the Arrow columnar format since version 1.0.0
    <post-1-0-0-format-versions>`
```

Comment:
Can you also add a subsection for version 1.5 in https://github.com/apache/arrow/blob/b4b22a4635fcf9bb1a36bf26411c6bde2ae01732/docs/source/format/Versioning.rst#post-100-format-versions ?

Reply:
added